src/components/ContextMenu/ContextMenuItem.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import ContextMenu from '.';
+
+function ContextMenuItem({ item, parentRef, isToLeft }) {
+  const [renderMenu, setRenderMenu] = useState(false);
+
+  function handleMouseEnter() {
+    setRenderMenu(true);
+  }
+
+  function handleMouseLeave() {
+    setRenderMenu(false);
+  }
+
+  switch (item.type) {
+    case 'item':
+      return (
+        <StyledItem className={`${item.inactive ? 'inactive' : ''}`}>
```

Comment:
Template literal isn't really used - this will work just fine: `className={item.inactive ? 'inactive' : ''}`

Reply:
Fixed.